src/uct/Makefile.am

```diff
@@ -33,7 +34,11 @@ noinst_HEADERS = \
 	sm/mm/base/mm_def.h \
 	sm/mm/base/mm_md.h \
 	sm/self/self.h \
-	tcp/tcp.h
+	tcp/tcp.h \
+	tcp/tcpcm/tcpcm_md.h \
```

Comment:
can we declare all needed things in the `tcp/tcpcm/tcpcm.h`?

Reply:
Why does the separation hurt? Most UCTs have this separation after all.

Reply:
> Why does the separation hurt? Most UCTs have this separation after all.

I agree